src/devices/vdml_usd.cpp

```diff
@@ -43,6 +43,19 @@ std::vector<std::string> list_files(const char* path) {
 			return files;
 		}
 	}
+
+	// Check path user passed in
+	std::string_view path_sv(path);
+	size_t found = path_sv.find("usd");
+	if (found == 0 || found == 1) {
+		// Deal with when user prepends path with usd
+		// as either "usd/..." or "/usd/..."
+		path_sv.remove_prefix(3);
+	}
+
+	// set path to path_sv.data()
+	path = path_sv.data();
```

Comment:
`path_sv` is already pointing to `*path`. It's a read-only "view" over a string that exists elsewhere in memory, hence the name.